### PR TITLE
refactor: refactor of issues in the UI

### PR DESCRIPTION
**Pull request by SebasthianSalpozz**
## Trello Ticket
https://trello.com/c/qDABcL7C

## Description

I work in problems that we see with the UI

## Type of Change

Put an `x` in all the boxes that apply:

- [ ] ✨ New Feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug Fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking Change (fix or feature that would cause existing functionality to change)
- [x] 🧹 Code Refactor
- [ ] ✅ Build Configuration Change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Pull Request Checklist

Put an `x` in all the boxes that apply:

- [ ] My commit messages are detailed.
- [x] My code follows the code style of this project.
- [ ] No existing features have been broken without good reason.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: LMS.DataAccess/src/Console/UserMenu/PatronMenu/Borrows/PatronBorrowsMenu.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ public void PatronBooksMenu()
                     break;
 
                 case "Return book":
-                    borrowAction = new BorrowAction();
+                    borrowAction = new BorrowAction();          AnsiConsole.Clear();
```
Comment (ignacio-villarroel-jalafund) [inline LMS.DataAccess/src/Console/UserMenu/PatronMenu/Borrows/PatronBorrowsMenu.cs]: Correct the method position

Comment (SebasthianSalpozz) [reply LMS.DataAccess/src/Console/UserMenu/PatronMenu/Borrows/PatronBorrowsMenu.cs]: Fixed :D

File: LMS.DataAccess/src/Console/UserMenu/PatronMenu/PatronMenu.cs [hunk 2]
```diff
@@ -51,16 +52,19 @@ public override void UserMenu()
...
                 case "Search Tool":
                     var searchMenu = new PatronSearchMenu(_books);
                     searchMenu.PatronSearchMenuOptions();
+                    Pause();
                     break;
 
                 case "Books List":
                     page.setPagination(new BookPagination(_books.GetBooksList()));
                     page.Execute();
+                    Pause();
```
Comment (ignacio-villarroel-jalafund) [inline LMS.DataAccess/src/Console/UserMenu/PatronMenu/PatronMenu.cs]: Remove this Pause() please

File: LMS.DataAccess/src/Console/UserMenu/PatronMenu/PatronMenu.cs [hunk 3]
```diff
@@ -51,16 +52,19 @@ public override void UserMenu()
                 case "Borrow Options":
                     var borrowMenu = new PatronsBorrowMenu(_borrows, _patron, _books, _fines);
                     borrowMenu.PatronBooksMenu();
+                    Pause();
                     break;
 
                 case "Search Tool":
                     var searchMenu = new PatronSearchMenu(_books);
                     searchMenu.PatronSearchMenuOptions();
+                    Pause();
```
Comment (ignacio-villarroel-jalafund) [inline LMS.DataAccess/src/Console/UserMenu/PatronMenu/PatronMenu.cs]: Remove this please

File: LMS.DataAccess/src/Console/UserMenu/PatronMenu/PatronMenu.cs [hunk 4]
```diff
@@ -51,16 +52,19 @@ public override void UserMenu()
                 case "Borrow Options":
                     var borrowMenu = new PatronsBorrowMenu(_borrows, _patron, _books, _fines);
                     borrowMenu.PatronBooksMenu();
+                    Pause();
```
Comment (ignacio-villarroel-jalafund) [inline LMS.DataAccess/src/Console/UserMenu/PatronMenu/PatronMenu.cs]: Remove this please

**Review by ignacio-villarroel-jalafund — APPROVED** [review]

**Pull request merged**